docs/infra/set-up-database.md

```diff
@@ -26,6 +26,29 @@ make infra-configure-app-database APP_NAME=<APP_NAME> ENVIRONMENT=<ENVIRONMENT>
 `APP_NAME` needs to be the name of the application folder within the `infra` folder. By default, this is `app`.
 `ENVIRONMENT` needs to be the name of the environment you are creating. This will create a file called `<ENVIRONMENT>.s3.tfbackend` in the `infra/app/service` module directory.
 
+### (Optional) Enable any database extensions that require `rds_superuser`
+
+To enable some extensions, such as [pgvector](https://github.com/pgvector/pgvector), requires the `rds_superuser` role. You can enable any such extensions via the `superuser_extensions` configuration variable, and set them to either enabled or disabled.
+
+For example, to enable the pgvector extension:
+
+```terraform
+# infra/app/app-config/env-config/outputs.tf
+
+output "database_config" {
+  value = var.has_database ? {
+    ...
+    superuser_extensions = {
+      "vector": true,
```

Comment:
non-blocking question: is it possible to see an existing list of superuser extensions within postgres e.g. something like select * from pg_extension where oid (ownerid) is the superuser? https://www.postgresql.org/docs/current/catalog-pg-extension.html

just wondering if that would allow this to be a simple list instead of a dictionary.

Reply:
I think this works but worry it would get messy -- there are often other extensions installed, e.g., `plpgsql`, so we'd probably want to add code to check that we weren't turning off anything RDS was including automatically just because it wasn't in this allow-list.